brokerapi/brokers/cloudsql/mysql-definition.go

```diff
@@ -29,174 +29,154 @@ const MySqlServiceId = "4bc59b9a-8520-409f-85da-1c7552315863"
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-standard-64 (64 CPUs, 240 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "e45d7c44-4990-4dac-a14d-c5127e9ae0c5",
-				        "name": "mysql-db-n1-standard-64",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-2",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-2 (2 CPUs, 13 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "07b8a04c-0efe-42d3-8b2c-2c23f7c79583",
-				        "name": "mysql-db-n1-highmem-2",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-4",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-4 (4 CPUs, 26 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "50fa4baa-e36f-41c3-bbe9-c986d9fbe3c8",
-				        "name": "mysql-db-n1-highmem-4",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-8",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-8 (8 CPUs, 52 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "6e8e5bc3-bf68-4e57-bda1-d9c9a67faee0",
-				        "name": "mysql-db-n1-highmem-8",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-16",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-16 (16 CPUs, 104 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "3c83ff6b-165e-47bf-9bba-f4801390d0ff",
-				        "name": "mysql-db-n1-highmem-16",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-32",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-32 (32 CPUs, 208 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "cbc6d376-8fd3-4a34-9ab5-324311f038f6",
-				        "name": "mysql-db-n1-highmem-32",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {
-				            "tier": "db-n1-highmem-64",
-				            "max_disk_size": "10230"
-				        },
-				        "description": "MySQL on a db-n1-highmem-64 (64 CPUs, 416 GB/RAM, 10230 GB/disk, 4,000 Connections)",
-				        "id": "b0742cc5-caba-4b8d-98e0-03380ae9522b",
-				        "name": "mysql-db-n1-highmem-64",
-				        "free": false
-				    }
-				]
-		}`,
+		Id:               "4bc59b9a-8520-409f-85da-1c7552315863",
+		Name:             "google-cloudsql-mysql",
+		Description:      "Google Cloud SQL is a fully-managed MySQL database service.",
```

Comment:
This also doesn't seem to match the diff
```
532c532
<                 "description": "Google Cloud SQL is a fully-managed MySQL database service.",
---
>                 "description": "Google CloudSQL for MySQL is a fully-managed MySQL database service.",
```

Reply:
Nice catch, it would help if I `push`ed all the updates before I completed the PR. :-)